src/Packages/UnityWebBrowser/Runtime/Core/RawImageUwbClientInputHandler.cs

```diff
@@ -89,13 +98,18 @@ private IEnumerator KeyboardAndMouseHandler()
 
             while (Application.isPlaying)
             {
-                if(!browserClient.IsConnected)
+                yield return 0;
+
+                if (!browserClient.IsConnected)
                     continue;
-                
+
+                if(disableAllControls)
```

Comment:
Since we not going to have `disableAllControls`, just check if both are disabled.